interface.go

```diff
@@ -38,6 +37,9 @@ type Bucket interface {
 	// implementation.
 	Check(context.Context) error
 
+	// Exists returns whether the given key exists in the bucket or not.
+	Exists(context.Context, string) (bool, error)
```

Comment:
does it make sense to add tests of this method?

Reply:
I added tests!

Reply:
Oh sorry, I missed that somehow.